Core/Core/View/Base/Webview/WebView.swift

```diff
@@ -120,7 +61,24 @@ public struct WebView: UIViewRepresentable {
             addObserver()
         }
 
+        public func webView(_ webView: WKWebView, didStartProvisionalNavigation navigation: WKNavigation!) {
+            webView.isHidden = true
+        }
```

Comment:
Is it important to hide the webview here? as hiding it here is required `webview?isHidded = true` at multiple places

Reply:
Is it possible to use `isLoading` property here, so the user will see loading while the inversion is happening?

Reply:
no, `isLoading` property doesn't hide webView it's shows `ProgressView` on parent view.

Reply:
Actually, that's what I'm trying to understand why it's needed, because if it's user-facing and there won't be any loading screen, and we also hide the webview, the view will not look good. If we se the isloading true, the user will see a loading screen and behind the scene the color inversion will happen.

Reply:
just to clarify, do you want to set `isHidden` when we set `isLoading`? If so then it will work.

Reply:
without `isHidden`:

https://github.com/openedx/openedx-app-ios/assets/480059/49f436f6-2f72-4608-8b16-44ac480d2e70

with `isHidden`

https://github.com/openedx/openedx-app-ios/assets/480059/7302a350-4e0e-44ab-8dff-30a6d4b0a161